runtime/Stdlib.res

```diff
@@ -100,6 +100,7 @@ async function main() {
 external import: 'a => promise<'a> = "%import"
 
 let panic = Error.panic
+let throw = Error.throw
```

Comment:
This is not zero-cost (causes a `$$throw` function to be emitted in Stdlib.js). I am afraid we need to repeat the `external` definition here.

Reply:
Done